pages/linux/cewl.md

```diff
@@ -0,0 +1,23 @@
+# CeWL
+
+> URL spidering tool for making a cracking wordlist from Web content.
+
```

Comment:
Can we insert a "more information" 2nd description line here at all?

Reply:
I've just taken a look, and I think [this](https://digi.ninja/projects/cewl.php)  might be the homepage for this tool?

In this case, what about this?
```suggestion
> More information: <https://digi.ninja/projects/cewl.php>.

```